pkg/operation/common/types.go

```diff
@@ -75,6 +75,24 @@ const (
 	// AlertManagerPrefix is a constant for a prefix used for the AlertManager instance.
 	AlertManagerPrefix = "au"
 
+	// LokiPrefix is a constant for a prefix used for the Loki instance.
+	LokiPrefix = "l"
+
+	// IngressPrefix is the part of a FQDN which will be used to construct the domain name for an ingress controller of
+	// a Shoot cluster. For example, when a Shoot specifies domain 'cluster.example.com', the ingress domain would be
+	// '*.<IngressPrefix>.cluster.example.com'.
+	IngressPrefix = "ingress"
+
+	// APIServerPrefix is the part of a FQDN which will be used to construct the domain name for the kube-apiserver of
+	// a Shoot cluster. For example, when a Shoot specifies domain 'cluster.example.com', the apiserver domain would be
+	// 'api.cluster.example.com'.
+	APIServerPrefix = "api"
+
+	// InternalDomainKey is a key which must be present in an internal domain constructed for a Shoot cluster. If the
+	// configured internal domain already contains it, it won't be added twice. If it does not contain it, it will be
+	// appended.
+	InternalDomainKey = "internal"
```

Comment:
Remove all of this and use https://github.com/gardener/gardener/blob/abf03988344cca36be87cb272558f9b53a72624d/pkg/utils/gardener/dns.go#L36-L47

Reply:
Thanks